testnets/_IBC/axelartestnet-xiontestnet.json

```diff
@@ -1,30 +0,0 @@
-{
-  "$schema": "../../ibc_data.schema.json",
-  "chain_1": {
-    "chain_name": "axelartestnet",
-    "client_id": "07-tendermint-970",
-    "connection_id": "connection-752"
-  },
-  "chain_2": {
-    "chain_name": "xiontestnet",
```

Comment:
There are assets registered that depend on IBC connections like these being registered. I think just changing the chain_name to "axelartestnet1" should suffice. (and in the file name)